docs/conf/atlantic/2023/news/announcing-schedule.rst

```diff
@@ -0,0 +1,80 @@
+
+Today we're announcing the full presentation schedule.
+
+Conference platform
+--------------------
+
+The conference will again be held on https://hopin.to/. Hopin not only provides a stage for our amazing speakers, but also has integrated chat, unconference rooms with breakout calls, and expo booths. 
+Our Welcome Wagon guide, to be released soon, will help it feel familiar before you've even arrived.
+Hopin will show you the conference schedule in your time zone, making it easy to follow the schedule.
+
+You will receive an invite to Hopin about one week before the conference.
+
+Full presentation schedule
+--------------------------
+
+The conference website is now updated with the :doc:`/conf/{{shortcode}}/{{year}}/schedule/` for the main stage presentations, so you can start planning your time (and getting excited!) for the conference. 
+We're excited to continue live Q&A after most talks. The Q&A will come from your questions but moderated by our MCs.
+
+As our attendees are in various time zones, we've planned a number of equal breaks throughout the day. We are all trying to balance our screen time, so we wanted to make sure that your eyes get their rest!
+
+Instead of a separate social event, we've added time in the schedule before the start and after the end of each day for you to socialize in the hallway sessions or one on one.
+
+**All talk videos will be published on our YouTube channel no later than 1 week after the conference.**
+
+Start thinking about Writing Day
+--------------------------------
+
+Our pre-conference `Writing Day <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/writing-day/>`_ will happen on Sunday, and it is much more fun when folks come with their own projects in mind!
+
+If you have a project or are part of a community that you think would benefit from a writing sprint, start brainstorming now about what you might want to focus on.
+If you'd like us to include your project on the Writing Day page `submit your project <https://forms.gle/KPo1ZPuRHqf7UZy37>`_ and we'll
+add it to our `Writing Day project list <https://www.writethedocs.org/conf/atlantic/{{year}}/writing-day/#your-project-here>`__.
+You can also just show up on Sunday ready to introduce your project, but getting it on the website in advance can increase interest.
+
+
+Shirts still on sale
+--------------------
+
+The `Write the Docs {{city}} {{year}} Pop-Up Shop <https://shirt.writethedocs.org/>`_ is open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want.
+
+The campaign will run in batches until September 30th. Shipping is available from the USA or Europe. However, due to the global shipping situation, we recommend you order your shirt at least 2-3 weeks in advance if you want to sport your new threads during the conference.
+
+All our events and activities
```

Comment:
I know we're not doing a job fair, but do we want to mention the Sponsor Expo and explain briefly what it is? cc @ericholscher

Reply:
Seems reasonable, I see for Portland we didn't do this, so not a huge deal, but likely worth mentioning? 🤷